README.md

```diff
@@ -111,38 +94,38 @@ A table containing any of the following keys may be passed into the Conctr const
 
 | Key | Data type | Default value | Description |
 | ----| --------------- | --------- | ----------- |
-| isEnabled | Boolean | `true` | When enabled, location data will be automatically included with the data payload|
-| interval | Integer | 3600000|  Duration in milliseconds since last location update to wait before sending a new location |
-| sendOnce | Boolean | `false` | Setting to `true` sends the location of the device only once when the device restarts |
+| send_loc | Boolean | `true` | When enabled, location data will be automatically included with the data payload|
```

Comment:
switched out of camel case as I had the contructor values (app_id, api_key etc) in that form and thats how they had it in the Readme.md i used as a base for mine, just assumed i should have one convention through out? Should i change everything to camelcase including the constructor?